src/OpenApi/gen/XmlCommentGenerator.Emitter.cs

```diff
@@ -383,6 +387,29 @@ public Task TransformAsync(OpenApiOperation operation, OpenApiOperationTransform
 
             return Task.CompletedTask;
         }
+
+        private static OpenApiParameter UnwrapOpenApiParameter(IOpenApiParameter sourceParameter)
+        {
+            if (sourceParameter is OpenApiParameterReference parameterReference)
+            {
+                if (parameterReference.Target is OpenApiParameter target)
+                {
+                    return target;
+                }
+                else
+                {
+                    throw new InvalidOperationException("The input schema must be an OpenApiSchema or OpenApiSchemaReference.");
```

Comment:
Copy-pasta? This is checking parameter not schema

Reply:
Good catch -- will fix.